cura/API/__init__.py

```diff
@@ -21,3 +24,9 @@ class CuraAPI:
 
     # Interface API
     interface = Interface()
+
+    _account = Account()
```

Comment:
Okay, so what do you propose? Moving it into the authenticationService won't actually solve that problem. Moving it to the oauth settings won't really decrease the problem, since then you suddenly need to add extra functionality to store the data somewhere.

Reply:
I've changed it so that there is now a weak link between preferences and the account.